docs/content/usages/promise.md

```diff
@@ -0,0 +1,87 @@
+---
+sort: 2
+---
+
+# Action Promise
+
+Nikita's actions always return [Javascript Promise](https://nodejs.dev/learn/understanding-javascript-promises). To access the action output or to run JavaScript commands synchronously, you have to call an asynchronous function and ["await"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await) for the result from Promise:
+
+```js
+// New Nikita session
+nikita
+// Call an action
+// highlight-range{1-2}
+.call(async function() {
+  const result = await this.call({
+    who: 'leon',
+    handler: ({config}) => {
+      return {who: config.who}
+    }
+  })
+  // Run next commands synchronously
+  console.info(result.who)  // leon
+})
+```
+
+To access the result of the global Nikita session, you have to wrap the code with the call of [anonymous asynchronous function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/async_function):
```

Comment:
wrong, wrapping the call has nothing to do with calling the nikita session, it is a node.js issue, not related to nikita, you can access the result of a nikita session normally if you are not at the top level, and only [until recently](https://pprathameshmore.medium.com/top-level-await-support-in-node-js-v14-3-0-8af4f4a4d478).

Reply:
changed